install_config/aggregate_logging.adoc

```diff
@@ -788,3 +788,63 @@ If you are not using the provided Kibana and Elasticsearch images, you will not
 have the same multi-tenant capabilities and your data will not be restricted by
 user access to a particular project.
 ====
+
+== Performing Elasticsearch maintenance operations
+
+As of the Deployer version
+ifdef::openshift-origin[]
+1.2.0
+endif::openshift-origin[]
+ifdef::openshift-enterprise[]
+3.2.0
+endif::openshift-enterprise[]
+an admin certificate, key, and ca that can be used to communicate with and perform
+administrative operations on Elasticsearch is provided within the
+`logging-elasticsearch` secret.
+
+[NOTE]
+====
+To confirm whether or not your EFK installation provides these, run:
+----
+$ oc get secret/logging-elasticsearch -o yaml | grep 'admin-.*:'
```

Comment:
IMHO this would be a lot clearer to read:

```
$ oc describe secret logging-elasticsearch
Name:           logging-elasticsearch
Namespace:      logging
Labels:         <none>
Annotations:    <none>

Type:   Opaque

Data
====
admin-cert:             1233 bytes
admin-key:              1704 bytes
key:                    4050 bytes
searchguard.key:        32 bytes
truststore:             817 bytes
admin-ca:               1078 bytes
```